CONTRIBUTING.md

```diff
@@ -1,5 +1,47 @@
 # Contributing
 
+## Ancillary Technologies
+This section serves as a reference for various supplementary technologies associated with Keptn. 
+The information has been gathered from the community and is subject to alteration. 
+It is suggested that this list be utilized as a means of gaining familiarity with the ancillary technologies of Keptn. 
+If you believe that additional content should be included in this list, please submit a comment on issue
+[#733](https://github.com/keptn/lifecycle-toolkit/issues/733).
```

Comment:
Better to tell them to file an issue.  #733 will be closed in a few weeks.

Reply:
updated accordingly